### PR TITLE
add windows guest agent installation method

### PR DESCRIPTION
**Pull request by Zhengtong**
id: 1340676

### CODE REVIEW

File: qemu/deps/windows_ga_install/get_package.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+                return line.split()[0]
+        raise Exception("didn't get latest build name")
+
+    def get_build_url(self):
+        build_name = self.get_latest_build()
+        cmd = 'brew buildinfo %s | grep msi' % build_name
+        (status, output) = self._run_shell_cmd(cmd)
+        url_list = []
+        for package in output.splitlines():
+            url = re.sub(r'/mnt/redhat', self.server_url, package)
```
Comment (xutian) [inline qemu/deps/windows_ga_install/get_package.py]: it looks 
--topdir
--brewurl
--brewhub/--server
should be include in your args. more details please see /etc/brewkoji.conf

Comment (Zhengtong) [inline qemu/deps/windows_ga_install/get_package.py]: Thanks for advice, updated

**Comment by xutian** [comment]
looks OK to me.

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by Xueqiang Wei <xuwei@redhat.com>

**Pull request merged**